tests/kani/Print/main.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// This test checks that println! and eprintln! do not result in verification
```

Comment:
```suggestion
// This test checks that `print!`, `println!`, `eprint!` and `eprintln!` do not result in verification
```
or just use "print macros" to refer to these

Reply:
Fixed.